GDJS/Runtime/layer.ts

```diff
@@ -238,6 +240,11 @@ namespace gdjs {
 
       // The result parameter used to be optional.
       let position = result || [0, 0];
+
+      if (this._renderer.isCameraRotatedIn3D()) {
```

Comment:
I'm a bit sad about this thing that mix the responsibility between the renderer and the layer.ts file. Maybe we can move the convertCoords at some point (and the other methods) to the renderer. Or keep all the logic here?

Reply:
Add a comment "TODO (3D): this should in theory be in a single file".

Reply:
Yes, there is probably a decision to make.
Should GDJS handle 3D transformations itself?
We did for 2D, but calculus are more complicated in 3D. It's probably something to leave to Three.js.